plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -241,6 +250,40 @@ func (c *nodePlugin) admitPodEviction(nodeName string, a admission.Attributes) e
 	}
 }
 
+func (c *nodePlugin) admitPVCStatus(nodeName string, a admission.Attributes) error {
+	switch a.GetOperation() {
+	case admission.Update:
```

Comment:
should forbid unless ExpandPersistentVolumes feature is enabled, right?

Reply:
fixed